app/services/user_service.rb

```diff
@@ -18,8 +18,8 @@ def self.search(email)
     JSON.parse(response.body, symbolize_names: true)
   end
 
-  def self.friends_of(user)
-    response = DatabaseService.conn(user).get("api/v1/friends")
+  def self.user_relationships_filter(user, request_status)
```

Comment:
This is nice, it seems to be a more flexible endpoint to get different types of relationships.